client/src/components/Project/index.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from "react";
+import { NavLink } from "react-router-dom";
+import { Button } from "antd";
+import Scrum from "./Scrums/index";
+import './style.css';
+const scrums = require('./Scrums/utilis/scrums.json')
+
+class Scrums extends Component {
+  state = {
+    project: {
+      id: '',
+      projectName: ''
+    },
+    scrums: []
+  }
+
+  componentDidMount(){
```

Comment:
If you noticed. When you add a task to a scrum and them navigate to another scrum, the new task will still appear. 
You should use componentDidUpdate. 
```javascript
componentDidUpdate(prevProp, prevState){
    if(prevProp.scrumId !==this.props.scrumId){
      this.setState({tasks});
    }
  }
```
You should keep in mind when you start working in the back-end side that you would fetch the new scrums' tasks from the database inside this lifecycle method and then update the state. Is that right, @ashatat ?